src/networking/serialization/mod.rs

```diff
@@ -97,6 +120,28 @@ mod tests {
     // doesnt have anything but needed for the default implementation (as of right now)
   }
 
+// only works with feature(test) and with `extern crate test; use test::Bencher`
```

Comment:
Should probably be removed and replaced in favor of proper benchmarks with `Criterion.rs`

Reply:
Doesn't necessarily have to be done right now though

Reply:
not doing right now. making an issue in the project board